test/scalability/default_rangespec.yaml

```diff
@@ -5,14 +5,8 @@
 # - #1782772615836864512
 # - #1782775995984515072
 cmd:
-  # Average value 351116.4 (+/- 0.9%), setting at +5%
-  maxWallMs: 368_000
-
-  # Average value 111500 (+/- 14%), setting at +20%
-  maxUserMs: 134_000
-
-  # Average value 27875 (+/- 16%), setting at +20%
-  maxSysMs: 34_000
+  # Average value 396 mCPU (+/- 8%), setting at +25%
+  mCPU: 500
```

Comment:
I actually prefer not to measure average cpu, because it is hard to get intuition about if lower or higher is actually better. The previous measurements are more intuitive. I guess if there are too meany of them to set, then
I would suggest to control just for `maxWallMs` as the most intuitive.

Reply:
I think from the user's perspective what is important is the overall time, and this is what we should primarly control for.

Reply:
I also don't see a direct relationship between CPU time and user-perceived performance.

If we are doing parallel algorithms, we would user more CPU and that's not necessarily bad.

Reply:
1. The difference when it comes to timing more or less direct will be detected by wl admission times or the runner's timeout if relevant. 

2. The CPU consumption matters also, I believe that the default deployment of kueue has 500nCPU allocated by default.

3. Checking absolute values of sys and user time is too much for our use-case.

Reply:
> Checking absolute values of sys and user time is too much for our use-case.

I agree with this. But then we can just keep the wall time.

Regarding CPU utilization, I agree with the following:

> it is hard to get intuition about if lower or higher is actually better.

Reply:
If doing the same job quantity (static by config) and quality (expectations are checked) less CPU is better.

Reply:
We might need to later optimize the algorithms using parallelism (and correspondingly increase the default CPU requests).

Once you have parallelism, CPU usage wouldn't account for how long we are waiting on locks.

Reply:
As long as the implementation requires extra resources (cpu, memmory, time) and we are aware of that there is no problem changing rangespec in the same pr.

Reply:
I've added back the wall time check, but when it comes to sys vs user time , we really should not care what the cpu is doing and continue to check the average mCPU consumption.

Reply:
I'm happy for dropping `UserMs` and `SysMS`. I would suggest to also not add `mCPUUsed` and I'm happy to go forward with this PR.

Reply:
> For example, my intuition tells me that in this https://github.com/kubernetes-sigs/kueue/pull/1952 we might have increased mCPU by avoiding waiting on List requests, and that is desired.

For this kind of case there is no problem to update the rangespec as well. 

The the reason of this checks should not be to set some absolute value to be respected but help us detect when we louse performance (having longer wait to admission times, having a drop in the cluster queue resource usage, using more memory, using more CPU) without a clear expectation for that to happen.

Reply:
Having any of the checks goes beyond the [design doc](https://docs.google.com/document/d/1iO8CGO6kgav2JL6RKYnNYvkeQIHpFqB-SQqJ7eUsjPs/edit?tab=t.0#heading=h.e5jhgh56ju14). I'm ok to accept some of them for exploration purposes, but I'm afraid having the additional check, especially not so intuitive will add a chore.

Can we please make this PR a quick fix bumping the thresholds, and have a separate discussion on including extra thresholds?

Reply:
The check is doing the exact same thing, the only difference being that we judge sys and user times together.

Reply:
I don't think we need the checks for sys time, user time or mCPU at all for now.

Reply:
Cpu is an important resource, just like memory, why should it be ignored?

Reply:
For now we don't have a way to reliably set the thresholds. In order to move forward I suggest:
1. adjust in this PR the thresholds according to https://github.com/kubernetes-sigs/kueue/issues/2066#issuecomment-2079053502, this should help
2. follow up and do a small script that will automate it
3. discuss the new measurements, including mCPU with threshold set based on the script

Reply:
That makes it less useful. Let's remove it.

Reply:
> That makes it less useful.

How?